chrome/content/zotero/standalone/standalone.js

```diff
@@ -715,6 +715,23 @@ const ZoteroStandalone = new function() {
 			);
 			return { remove: result === 0, report: null };
 		};
+		// A11y: when a popup appears, mark which buttons are checked for screen readers
+		doc.addEventListener("shown", (event) => {
+			for (let item of [...event.target.querySelectorAll("panel-item")]) {
+				item.button.setAttribute("role", "menuitemcheckbox");
+				item.button.setAttribute("aria-checked", item.checked);
+			}
+		}, true);
```

Comment:
Does 33 happen in Firefox (115 or current)? We shouldn't have to set this manually every time we use a checked menu item.

Reply:
Yes, I'm on firefox 129.0 and if I go to `about:addons`, click on the "Tools for all add-ons" and tab to a checked "Update Add-ons Automatically", it still does not say that it's checked.

This has not been an issue for all menu items. All other regular menuitems within menus that I tried behave fine. For instance, top-level menus `View > *`, tag selector "Actions" menu, quickSearch mode selector - their checked  menuitems are announced as checked. So I think it's just this menu, and I suppose because it's a `panel-item`, and not a `menuitem` component that we use.

I am not sure if the "Throughout the Zotero desktop app" part of the vpat 33 comment implies that there are other instances of this happening. But I assume that if this were to happen elsewhere, it would have been brought up in other issues.